File: java/program/Pemilik.java
public class Pemilik extends Manusia {
    private Bengkel bengkel;

    public Pemilik(String nama, String alamat, Bengkel bengkel) {
        super(nama, alamat);
        this.bengkel = bengkel;
    }

    public Bengkel getBengkel() {
        return bengkel;
    }

    public void setBengkel(Bengkel bengkel) {
        this.bengkel = bengkel;
    }
}

File: cpp/program/Pemilik.cpp
#include "Pemilik.h"

Pemilik::Pemilik(string nama, string alamat, Bengkel* bengkel)
    : Manusia(nama, alamat), _bengkel(bengkel) {}

Bengkel* Pemilik::getBengkel() { 
    return _bengkel; 
}
